drivers/ieee802154/ieee802154_dw1000_regs.h

```diff
@@ -1925,40 +1925,29 @@ const uint8_t dwt_pgdelay_defs[] = {
 	DWT_TC_PGDELAY_CH7
 };
 
-/*
- * Defaults from Table 19: Reference values for Register file:
- *     0x1E – Transmit Power Control for Smart Transmit Power Control
- *     Transmit Power Control values for 16 MHz, with DIS_STXP = 0
- */
-const uint32_t dwt_txpwr_stxp0_16[] = {
-	0x15355575,
-	0x15355575,
-	0x0F2F4F6F,
-	0x1F1F3F5F,
-	0x0E082848,
-	0x32527292
-};
+#ifdef CONFIG_ENABLE_MANUAL_TX_POWER_CONTROL
```

Comment:
Please do not add any conditions to this header, this will need to be undone later to support multiple instances.